django/http/multipartparser.py

```diff
@@ -103,6 +105,8 @@ def __init__(self, META, input_data, upload_handlers, encoding=None):
         self._encoding = encoding or settings.DEFAULT_CHARSET
         self._content_length = content_length
         self._upload_handlers = upload_handlers
+        # Number of bytes that have been read.
+        self._data_size = 0
```

Comment:
This one would still make marginally more sense as a variable in the `parse` method, right?